index.html

```diff
@@ -1281,22 +1287,23 @@ <h3>Types</h3>
 
         <p>
 <a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST have a
-<code>type</code> <a>property</a>; in other words, any <a>credential</a> or
-<a>presentation</a> that lacks a <code>type</code> <a>property</a> <em>is not
-verifiable</em>, so is not a <a>verifiable credential</a> nor a
+<code>type</code> <a>property</a>. That is, any <a>credential</a> or
+<a>presentation</a> that does not have <code>type</code> <a>property</a>
+<em>is not <a>verifiable</a></em>, so is not a <a>verifiable credential</a> or a
```

Comment:
`is not a <a>verifiable credential</a> or a`
-> `is neither a <a>verifiable credential</a> nor a`

Reply:
Done.